From 41504f9d0f6e8a1232e1220da2b22eca0c393eb5 Mon Sep 17 00:00:00 2001
From: Sharlatan Hellseher <sharlatanus@gmail.com>
Date: Mon, 13 Mar 2023 23:06:47 +0000
Subject: [PATCH 07/17] gnu: cl-parachute: Update to 1.5.0-1.bd072b0.

* gnu/packages/lisp-check.scm (sbcl-parachute): Update to 1.5.0-1.bd072b0.
---
 gnu/packages/lisp-check.scm | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/gnu/packages/lisp-check.scm b/gnu/packages/lisp-check.scm
index fb65a27e62..172f5b3873 100644
--- a/gnu/packages/lisp-check.scm
+++ b/gnu/packages/lisp-check.scm
@@ -718,8 +718,8 @@ (define-public cl-nst
   (sbcl-package->cl-source-package sbcl-nst))
 
 (define-public sbcl-parachute
-  (let ((commit "8bc3e1b5a1808341967aeb89516f9fab23cd1d9e")
-        (revision "0"))
+  (let ((commit "bd072b0e4d0ff3ee2201eca3eb28c287686ab49e")
+        (revision "1"))
     (package
       (name "sbcl-parachute")
       (version (git-version "1.5.0" revision commit))
@@ -732,7 +732,7 @@ (define-public sbcl-parachute
            (commit commit)))
          (file-name (git-file-name "cl-parachute" version))
          (sha256
-          (base32 "0cppp1sp9xqkgxgkwidhqzlsj03ywnar7z9mzwcliww8y0kv5555"))))
+          (base32 "0srjsklhr04snlv98021mb0a5lb8dlypv1lnjdhsdhjbbiwbw2n9"))))
       (build-system asdf-build-system/sbcl)
       (inputs
        (list sbcl-documentation-utils
-- 
2.39.2

